parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -170,13 +170,20 @@ $global:EnableHostsConfigAgent = [System.Convert]::ToBoolean("{{ EnableHostsConf
 # These scripts are used by cse
 $global:CSEScriptsPackageUrl = "{{GetVariable "windowsCSEScriptsPackageURL" }}";
 
+# These windows nvidia gpu driver are used by windows cse
+$global:GpuDriverCudaUrl = "{{GetVariable "windowsGpuDriverCudaUrl" }}";

```

Comment:
to do: these urls parameters are supposed to be passed from rp the same way as CSEScriptsPackageUrl. The rp side changes have not been fully implemented.